MetaMask/metamask-extension
# feat: add B3 logo svg

## **Description**

Adding logo for the B3 network

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/27778?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

Go to add network
1. Add B3 network (https://docs.b3.fun)
2. B3 network should have a logo
3. B3 token should have a logo

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
